### PR TITLE
Corrected some words

### PR DESCRIPTION
**Pull request by desturi**
@rsura - Made some changes. Can you take a look at it?

### CODE REVIEW

File: Jasmin.txt [hunk 1]
```diff
@@ -2,12 +2,12 @@ Do you want to learn how to commit github?
...
 There are not many things in the file. So, just adding different flowers:
 - Rose
-- Jasmine
+- Jasmin
 - Carnation
 - Tulip
 
 
-Yes, I am interested and I am happy to be part of this team!  Good luck
+Yes, I am intherested and I am happy to be part of this team!  Good luck
```
Comment (rsura) [inline Jasmin.txt]: Looks like this spelling is bad, can you please correct it :+1:

**Comment by desturi** [comment]
@rsura - I have made the changes that you recommended

**Comment by rsura** [comment]
si, que bueno :+1:

**Comment by rsura** [comment]
ok

**Pull request merged**